test/src/api-action-creator-tests.js

```diff
@@ -70,6 +70,33 @@ describe('APIActionCreators', function () {
         aac.doThing();
     });
 
+    it('should expose headers in string and hash form', function (done) {
+        var aac = new APIActionCreator({
+            displayName: 'api' + Math.random(),
+            doThing: {
+                route: '/mirror',
+                method: 'POST',
+                createRequest: function () {
+                    return {
+                        query: {
+                            contentType: 'application/json'
+                        }
+                    };
+                },
+                handleSuccess: function (req, res) {
+                    res.rawHeaders.should.be.type('string');
+                    res.headers.should.be.type('object');
```

Comment:
I can try when I have time. I just copied the tests from that file and similar tests just test the type